ui/app/i18n/registration/locale_en.json

```diff
@@ -121,5 +121,6 @@
     "FILE_UPLOAD_MUST_BE_LESS_THAN": "File size should be less than",
     "PATIENT_ATTRIBUTE_GIVEN_NAME_LOCAL": "First Name in Arabic",
     "PATIENT_ATTRIBUTE_FAMILY_NAME_LOCAL": "Last Name in Arabic",
-    "PATIENT_ATTRIBUTE_MIDDLE_NAME_LOCAL": "Middle Name in Arabic"
-}
\ No newline at end of file
+    "PATIENT_ATTRIBUTE_MIDDLE_NAME_LOCAL": "Middle Name in Arabic",
+    "PRE_PATIENT_LABEL": "Patient"
```

Comment:
@kavitha-sundararajan should we rename this from PRE_PATIENT_LABEL  possible other names as we discussed.

Reply:
updated as discussed @deepthi-mantena